### PR TITLE
Refactors slime potions.

### PR DESCRIPTION
**Pull request by harmonyn**
Changes in this PR so far: 
- Slime potions, rather than being a scattered mishmash of paths, are all under /obj/item/slimepotion.
- All slime potions have a weight class of 1. This fixes one of my most hated bugs: potions will now fit in your pockets, which they previously did not. REJOICE, XENOBIOLOGISTS, YOUR MOST ANNOYING MICRO IS DEAD. 
- Added an attack message to all slime potions when used on reagent containers to clarify for users that they must be used directly on slimes/mobs.
- Removed a redundant and never used potion definition. Nothing references it, it's exactly the same as something earlier, and Kor isn't here, so fuck it.

THE CONTROVERSIAL(?) STUFF STARTS BELOW
- Slime extracts have a biotech origin of 3, 1 less than before.
- Slime potions have a biotech origin of 4. Discourages: R&D murdering roundstart slimes for research before xenobiology can do shit. Encourages. Interdepartmental cooperation. You can get slime potions off two tier-2 slimes (blue and purple) so in practice this won't be too bad.
- Sentience potions have a biotech origin of 5. I play xenobiology a lot and this is basically the top of the xenobio tech tree so I can't imagine this breaking the balance too much.
- Slightly changed fluff text for how slime potions work. It is 100% utter and complete bullshit intended to write around the fact that slime potions do not work like actual potions (yet?) but for now it will do.

### CODE REVIEW

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -139,7 +148,7 @@
 	M.docile = 1
 	M.nutrition = 700
 	M <<"<span class='warning'>You absorb the potion and feel your intense desire to feed melt away.</span>"
-	user <<"<span class='notice'>You feed the slime the potion, removing it's hunger and calming it.</span>"
+	user <<"<span class='notice'>You feed the slime the potion, removing its hunger and calming it.</span>"
```
Comment (duncathan) [inline code/modules/research/xenobiology/xenobiology.dm]: <3 <3 <3 <3

**Comment by Core0verload** [comment]
Nice. I wanted to refactor that shit to reduce path clutter anyway.  :+1: 

But double-check RnD stuff. If it's possible to get xray using stuff at biotech 6, do not add that.

**Comment by bgobandit** [comment]
IIRC x-ray is biotech 5, nothing's at or provides biotech 6, alium brains provide 7

**Comment by GunHog** [comment]
Make no mistake that you will be allowing Science to make Xray implants by bypassing the need for alien brains/the radioactive microlaser/abductor items. Science would still have strong reasons to breed aliens either way, as we can steal their powers either way.

I am not against this, due to the items being gated so harshly in my opinion, but you should be aware of how it affects things, and if it matches with your intent.

**Comment by bgobandit** [comment]
wait x-ray implants too? goddamnit wiki you lied to me

how about this: normal slime potions are biotech 4 (same as implants), sentience potions are biotech 5. in order to get a sentience potion you need to breed gray ==> blue ==> pink ==> light pink, and you normally don't get there until round end

**Comment by SconesC** [comment]
as long as you can't make xray implants this is fine

**Pull request merged**